pyproject.toml

```diff
@@ -27,6 +27,7 @@ dependencies = [
     "spurplus ~= 2.3.5",
     "websockets ~= 10.3",
     "charset_normalizer ~= 2.1.1",
+    "requests ~= 2.32.0",
```

Comment:
Which package referred it? There is one on `ado` part, please also add version for it.

Reply:
it's referred by adal; and adal is referred by msrestazure.

added for the one in ado part

Reply:
`msrestazure` is in `azure`, so this should be added in the `azure` part.